docs/crocks/Identity.md

```diff
@@ -0,0 +1,61 @@
+# Identity
+
+`Identity a`
+
+Crock which returns the same value that was used as its argument
+
+## Why?
+
+To expose the constructor and instances defined below
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of) | [`ap`](#ap), [`chain`](#chain), [`equals`](#equals), [`map`](#map), [`of`](#of), [`sequence`](#sequence), [`traverse`](#traverse), [`value`](#value) |
+
+## Constructors
+
+### of
+
+`Identity a => a -> Identity a`
+
+## Instances
+
+### inspect
+
+`Identity a => Identity a`
+
+### value
+
+`() -> a`
+
+### type
+
+`() -> Identity`
+
+### equals
+
+`a -> Boolean`
+
+### map
+
+`Identity m => m (a -> b) -> m b`
+
+### ap
+
+`Identity m => m a ~> m (a -> b) -> m b`
```

Comment:
I don't know what you feel that.

There is a constraint that the instance that this is being called on, MUST contain a function.
So that is why `m (a -> b)` is needed to the left of `~>`...We need to communicate that constraint.

Make sense? Do you have any ideas on how we could better communicate that constraint?
👂